openshift/scripts/oc_deploy.sh

```diff
@@ -48,6 +48,7 @@ fi
 eval "${OC_PROCESS}"
 eval "${OC_PROCESS} | ${OC_APPLY}"
 eval "${OC_DEPLOY}"
+sleep 1 # Prevent timeout
```

Comment:
I wonder if it's possible to put a timeout on the oc log step to work around this?